src/baseai.ts

```diff
@@ -0,0 +1,54 @@
+import Ain from "@ainblockchain/ain-js";
+import Ainize from "@ainize-team/ainize-sdk";
+import Ainft721Object from "./ainft721Object";
+import { Path } from "./constants";
+import { buildTransactionBody } from "./util";
+
+export default class BaseAI {
+  private ain: Ain;
+  private ainize: Ainize;
+
+  constructor(ain: Ain, ainize: Ainize) {
+    this.ain = ain;
+    this.ainize = ainize;
+  }
+
+  async config(ainftObjectId: string, aiName: string): Promise<any> {
```

Comment:
Why here Promise < any >?

Reply:
Done! I replaced `any` to `TransactionResult` type.